packages/components/pie-toast/src/toast.scss

```diff
@@ -1 +1,72 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+.c-toast {
+  --toast-border-radius: var(--dt-radius-rounded-b);
+  --toast-background-color: var(--dt-color-container-inverse);
+  --toast-font-color: var(--dt-color-content-inverse);
+  --toast-font-size: #{p.font-size(--dt-font-body-s-size)};
+  --toast-line-height: #{p.line-height(--dt-font-body-s-line-height)};
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: 48px;
+  max-height: 122px;
+  min-width: 300px;
+  max-width: 420px;
+  
+  padding: var(--dt-spacing-a) var(--dt-spacing-c) var(--dt-spacing-a) var(--dt-spacing-d);
+  border-radius: var(--toast-border-radius);
+  gap: var(--dt-spacing-a);
+  background-color: var(--toast-background-color);
+  box-shadow: var(--dt-elevation-03);
+  color: var(--toast-font-color);
+  font-size: var(--toast-font-size);
+  line-height: var(--toast-line-height);
+
+  &-content-area{
```

Comment:
Naming thing – we use dashes to indicate child hierarchy of elements, so this would be `c-toast-contentArea` (and below would be `c-toast-messageArea`)